File: MyClub/MyClub.Model/Responses/AuthResponse.cs
using System;

namespace MyClub.Model.Responses
{

    public class AuthResponse
    {
        public int UserId { get; set; }
        public string Token { get; set; }
        public int RoleId { get; set; }
        public string RoleName { get; set; }
    }
}

File: MyClub/UI/myclub_mobile/lib/models/auth_response.dart
class AuthResponse {
  final int userId;
  final String token;
  final int roleId;
  final String roleName;

  AuthResponse({
    required this.userId,
    required this.token,
    required this.roleId,
    required this.roleName,
  });

  factory AuthResponse.fromJson(Map<String, dynamic> json) {
    return AuthResponse(
      userId: json['userId'],
      token: json['token'],
      roleId: json['roleId'],
      roleName: json['roleName'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'userId': userId,
      'token': token,
      'roleId': roleId,
      'roleName': roleName,
    };
  }
}
